core/aws_ddk_core/stages/sqs_lambda.py

```diff
@@ -36,8 +36,8 @@ def __init__(
         scope: Construct,
         id: str,
         environment_id: str,
-        code: Code,
-        handler: str,
+        code: Optional[Code] = None,
+        handler: str = "index.lambda_handler",
```

Comment:
Perhaps just `Optional[str] = None` like for Code? Even if we were to pass `index.lambda_handler` as a default, it would fail because Code is None

Reply:
It should stop complaining if you add an `if code and handler:` or an `assert`

Reply:
Tbh I also prefer not having a default for the handler, feel like this one user should be super conscious about